.github/workflows/docs-preview-prepare.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     paths:
       - 'docs/**'
+      - '.github/workflows/scripts/docs/build-docs.sh'
+      - '.github/workflows/docs-preview-prepare.yml'
+      - '.github/workflows/docs-preview-deploy.yml'
```

Comment:
I don't think `docs-preview-deploy.yml` will matter. `workflow_run` should always be run from master version not anything contributed in a PR. So starting a workflow based on changes to that file is likely going to be misleading.

`docs-preview-prepare.yml` requires no secrets, and triggered the `workflow_run` event.

```suggestion
```